consumer/doc.go

```diff
@@ -17,36 +17,35 @@
 // topic, and only the final one will count.  A typical call to Handle
 // looks like this:
 //
-//    c.Handle("testmsg", consumer.MessageConverterV1(cfg), HandlerFunc(func(m *consumer.Message) error {
+//    c.Handle("testmsg", consumer.MessageConverterV1(nil), HandlerFunc(func(m *consumer.Message) error {
 //        // Do something of your choice here!
 //        return nil // .. or return an actual error.
 //    }))
 //
-// Once you've registered all your handlers you may call Consumer.Serve.
+// Once you've registered all your handlers you should call Consumer.Serve.
 //
 // Serve will start consuming messages and pass them to their per-topic
-// handlers. Serve itself will block until Consumer.Stop is called.
+// handlers. Serve itself will block until Consumer.Close is called.
 // When Serve terminates it will return an error, which will be nil
 // under normal circumstances.
 //
 // Note that any calls to Consumer.Handle after
-// Consumer.Serve has been called will have no effect.
+// Consumer.Serve has been called will panic.
 //
-// The Handler interface defines the signature for all felice
-// Handlers. There are two common ways to comply with this interface.
-// The first is simply to create a type with the HandleMessage
-// function:
+// The Handler interface defines the signature for felice message
+// handlers. There are two common ways to comply with this interface.
+// The first is simply to create a type with the HandleMessage function:
 //
 //    type MyFooHandler struct { }
 //
-//    func (mfh MyFooHandler) HandleMessage(msg *consumer.Message) error {
-//        fmt.Printf("%+v", *msg)
+//    func (mfh MyFooHandler) HandleMessage(ctx context.Context, m *Message) error {
+//        fmt.Printf("%+v", msg)
 //    }
 //
 // The second approach is to cast a function to the HandlerFunc type
 // defined in this package:
 //
-//    h := handler.HandlerFunc(func(msg *consumer.Message) error {
-//        fmt.Printf("%+v", *msg)
+//    h := handler.HandlerFunc(func(ctx context.Context, m *Message) error {
+//        fmt.Printf("%+v", msg)
 //    })
 package consumer
```

Comment:
As we're shifting responsibility for error handling and retry to then user now, perhaps we should add some indications of how that is achieved in the docs here.